### ISSUE TITLE
Albums listed as by Various Artists despite editing tags to state otherwise

### ISSUE THREAD

**Issue by sabret00the** [clementine-player/Clementine#4714]
I have a few albums that Clementine seems to think are by Various Artists despite me editing the tags to state otherwise.

**Comment by chtfn** [comment]
I agree this is an issue, as far as I understand it.
I use "advanced grouping" for my library, with the first level as "Album artist". However, there are still releases listed in "Various artist" even though all the tracks have the same album artist tag.

**Comment by stragu** [comment]
Should this be marked as a bug, as the function "Advanced grouping > Group library by... First level = Album artist" does not work as expected?

**Comment by Chocobozzz** [comment]
Did you try the "don't show in various artist" right click option?

**Comment by stragu** [comment]
@Chocobozzz  Yes, I know about "don't show in various artists", it works fine, but it is just a workaround at the moment. The issue here is that Clementine still puts releases that have an "Album artist" value in "Various artists" when the first level of advanced grouping is set to "Album artist".

**Comment by stragu** [comment]
Happy to look into this if you can give me some pointers on Clementine's sorting system, as I know nothing about Clementine's code. This issue really bugs me :)

I am currently using 1.3.1 on KXStudio 14.04

**Comment by padre629** [comment]
I have the same issue in 1.3.1 Ubuntu 16.04 x64. Following the advice of @Chocobozzz was sufficient for me, thank you so much :+1: 

But I think ** "various artists" showing should be removed** or at least selectable by user in settings (disabled by default). I don't consider this feature as usable.

**Comment by jerobarraco** [comment]
> But I think "various artists" showing should be removed or at least selectable by user in settings (disabled by default). I don't consider this feature as usable

I totally agree, i wish this feature could be completely disabled. I'm managing a very big library and is really making it all harder.

I make changes or do a rescan often, and every time i have to do the "dont show in " again, and each time it takes more than 5 minutes to do so....

**Comment by Laurencio** [comment]
"Various Artists" is very annoying, it's mad thing.
why does this happen?
I i click on "Don't show on Various artists", after some time the tracks are again on it.
Is the "compilation flag" involved on this issue. If so, then "Don't show in various artists" should uncheck it (value as 0) and write this on files. 
If this is hard to achieve, then disable the hole Various artist thing.

**Comment by sellige00** [comment]
I have the samed problem (Windows 11 Clementine 1.3.1
When I  "Don't show in various artists", the rescan the library, the tracks are again in Various Artists

I don't understand  how and which tracks are set in "Various Artists"
